api/capi/include/nntrainer.h

```diff
@@ -27,7 +27,11 @@
 #include <stdbool.h>
 #include <stddef.h>
 
-#include <tizen_error.h>
+#if defined(__TIZEN__)
+#include <nnstreamer/ml-api-common.h>
+#else
+#include "ml-api-common.h"
+#endif
```

Comment:
Be careful, Tizen API auditors won't like this. 

Recommendation:
Use ```#include <ml-api-common.h>``` (nnstreamer.pc has ```Cflags: -I${includedir}/nnstreamer```)
Remove ```#if / #endif```
Update meson.build to set api/capi/include as includedir (allow to use ```<ml-api-common.h>``` for this directory) if it is not Tizen and try to disable it (move away or rename it) if it is Tizen.

Reply:
Thanks, it is fixed. Please take another look :)